examples/digital_fingerprinting/production/mlflow/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt update && \
     rm -rf /var/cache/apt/* /var/lib/apt/lists/*
 
 # Install python packages
-RUN pip install "mlflow<1.29.0" boto3 pymysql pyyaml
+RUN pip install "mlflow>1.29.0" boto3 pymysql pyyaml
```

Comment:
The `<1.29` was intentional to avoid a bug. I'm assuming the change to `>` is because its fixed in 1.30+?